Adding community fork section (#407)
voxpupuli/voxpupuli.github.io: _support/betadots.md

---
logo: sponsor-betadots.png
website: https://www.betadots.de/
button: betadots.de
---

> Betadots supports the majority of Puppet and OpenVox users in Germany.

Reviewer: ```suggestion
betadots supports many Puppet and OpenVox users in EMEA region.
```

> Betadots supports the majority of Puppet and OpenPuppet users in Germany. They

Reviewer 2: in the past we tried to wrap text after 80 chars in markdown. this gets problematic when a word is added or removed. Often you need to reformat a whole section and that makes the diff quite ugly. in Newer documents we try to have a single sentence per line, that seems to work better for reviews.

Reviewer 3: https://sembr.org/ is at least an interesting read

Author: I don't have strong opinions about sembr. The modules team switched to it a while back and it was fine. It's about the same amount of effort either way. You're either relying on your editor to soft-wrap in a comfortably readable way or relying on git/github to ignore whitespace for diffs.

Of course, soft-wrap is customizable to your own preferences, so maybe that's better?

I can change it if you want. Just gotta switch off the auto-formatter.

Reviewer 3: I agree it's not perfect but in my experience it has been easier in PR reviews to comment on specific pieces of text. I also felt it produced better diffs but that's just a general feeling. So a slight preference but not strong enough to push for it

> They offer consulting plans for both Puppet Enterprise and OpenVox.

Reviewer 2: ```suggestion
They offer training, custom development and consulting plans for both Puppet Enterprise and OpenVox.
```
